filesystems/drs/src/test/scala/cromwell/filesystems/drs/DrsPathBuilderSpec.scala

```diff
@@ -99,8 +99,8 @@ class DrsPathBuilderSpec extends TestKitSuite with FlatSpecLike with Matchers wi
       description = "a path ending in /",
       path = s"dos://$bucket/hello/world/",
       normalize = false,
-      pathAsString = s"dos://$bucket/hello/world/",
-      pathWithoutScheme = s"$bucket/hello/world/",
+      pathAsString = s"dos://$bucket/hello/world",
+      pathWithoutScheme = s"$bucket/hello/world",
```

Comment:
The name of this spec is `description = "a path ending in /"`, but that's no longer what it's doing.

Also, should `s"$bucket/hello/world/"` be tested, either as an expected Good or Bad path?

Reply:
So if we end up doing the extra sanitization step I mentioned in the comment above (removing all trailing slashes before we form a DRS path), even if the given input is `dos://abc/path/` it will be considered as `dos://abc/path`. So since paths with a `/` at the end would be considered a valid aka Good path (no ?) I left the test case but updated the `pathAsString` and `pathWithoutScheme` to reflect what the actual DRS path would look like.

Reply:
Aha, thanks!